packages/vscode-extension/package.nls.json

```diff
@@ -44,7 +44,7 @@
     "teamstoolkit.codeFlowLogin.loginTimeoutDescription": "Timeout waiting for login. Try again.",
     "teamstoolkit.codeFlowLogin.loginTimeoutTitle": "LoginTimeout",
     "teamstoolkit.codeFlowLogin.resultFileNotFound": "Result file not found.",
-    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently: %s",
+    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently. If you encounter this problem multiple times, you can try to delete `%s`, close all VSCode instances and try again. %s",
```

Comment:
you can try to delete -> you can delete
VS Code -> Visual Studio Code

Reply:
Done.